config/schedule.rb

```diff
@@ -26,7 +26,7 @@
 # Grant ETL
 every 2.months do
   exe 'exe/transform call StanfordPeopleList -i data/researchers.ndj > data/researchers.csv ' \
-    '&& exe/grants load -s 3 -i data/researchers.csv '
+    '&& exe/grants load -s 3 -i data/researchers.csv -d data/raw/grants -o data/grants'
```

Comment:
do you need to make a `data/raw/grants/.keep` file and add an entry into `.gitignore`?

Reply:
I don't know. I hand-added these to `./shared/` on the prod server, but letting the repo handle this seems more foolproof. Change incoming.

Reply:
Done.